pebbles/templates/invitation.txt

```diff
@@ -1,4 +1,4 @@
-You've been invited to participate in a Pebbles instance.
+You have been invited to participate in "notebooks.csc.fi".
```

Comment:
Like said in Flowdock, this is a case of hardcoding notebooks to the Pebbles codebase. We want to add another variable into the call that treats this as a template string and reference the variable in invitation.txt.

I think INSTALLATION_NAME is the best config variable to use in this case. Consider adding INSTALLATION_DESCRIPTION as well if you think it makes the email more readable.

Oh, and participate in --> use.